docs/first-steps/aws-account-setup.md

```diff
@@ -0,0 +1,27 @@
+# Set Up your AWS Management account
+
+## Create an AWS account
+First and foremost you'll need to [create an AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) for your project's deployment, this will be the management (Organizations' root) account for your project.
```

Comment:
How about rephrasing this:
"this will be the management (Organizations' root) account for your project."
...to this:
"this will be the root user of the management account of your AWS Organization." ?

Reply:
@diego-ojeda-binbash OJ I think that @angelofenoglio refers specifically to the management account (before called root) creation. No the root user.